### PR TITLE
Add contact info in footer

### PR DESCRIPTION
**Pull request by albandum**
## Description

Add contact mailto in the footer
Screenshot 2024-05-22 at 11 26 21

### CODE REVIEW

**Comment by Duncid** [comment]
Can you share a screenshot?

**Comment by albandum** [comment]
> Can you share a screenshot?

Added to description

**Review by spolu — APPROVED** [review]

File: front/components/home/menu/config.ts [hunk 1]
```diff
@@ -34,6 +34,11 @@ const CareersMenuConfig: MenuConfig = {
 const AboutMenuConfig: MenuConfig = {
   title: "About",
   items: [
+    {
```
Comment (flvndvd) [inline front/components/home/menu/config.ts:37]: IIRC, This will also add it in the main navigation menu at the top.

Comment (albandum) [reply front/components/home/menu/config.ts:37]: Screenshot 2024-05-22 at 12 11 39
 LGTM as well - it moved the x link out of the header, but as it's still available in the footer I'd say it's fine

Comment (Duncid) [reply front/components/home/menu/config.ts:37]: Would be better to not have it on top no?

Comment (albandum) [reply front/components/home/menu/config.ts:37]: Le me know where and I'll move 
Right below "blog"? If I move belove the X link it won't appear in the header. I don't have a specific pref.

**Pull request merged**